admin-ui/app/routes/Apps/Gluu/GluuRemovableInputRow.tsx

```
@@ -64,9 +93,9 @@ function GluuRemovableInputRow({
         )}
         <div
           role="button"
-          style={applicationStyle.removableInputRow as any}
-          onKeyDown={handler}
-          onClick={handler}
+          style={applicationStyle.removableInputRow as React.CSSProperties}
+          onKeyDown={() => handler()}
+          onClick={() => handler()}
         >
```

Biome (2.1.2)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)